apps/api/src/app/domain-modules/user-management.module.ts

```diff
@@ -31,6 +33,9 @@ import { CoscradUserController } from '../controllers/coscrad-user.controller';
         AddUserToGroupCommandHandler,
         GrantUserRole,
         GrantUserRoleCommandHandler,
+        // We include this command here for lack of a better place
```

Comment:
Is there another location for it down the road you have in mind?

Reply:
Not really. Each resource type has its own module. Yet this needs to be generic over all resource types. Unless we want a dedicated 'Generic-resource-workflow' module, I can't imagine where this belongs.